<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Themewidget</class>
 <widget class="QWidget" name="Themewidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255,0);
font: 700 36pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>  个性主题</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>370</width>
           <height>220</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>370</width>
           <height>220</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="111.qrc">
           <normaloff>:/icon/109.png</normaloff>:/icon/109.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>500</width>
           <height>210</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>370</width>
           <height>220</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>370</width>
           <height>220</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);
border-radius:10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="111.qrc">
           <normaloff>:/icon/108.png</normaloff>:/icon/108.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>400</width>
           <height>210</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="111.qrc">
         <normaloff>:/icon/1101.jpg</normaloff>:/icon/1101.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="111.qrc">
         <normaloff>:/icon/1102.jpg</normaloff>:/icon/1102.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="111.qrc">
         <normaloff>:/icon/1103.jpg</normaloff>:/icon/1103.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>250</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="111.qrc">
         <normaloff>:/icon/1104.jpg</normaloff>:/icon/1104.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="111.qrc">
         <normaloff>:/icon/1105.jpg</normaloff>:/icon/1105.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="111.qrc">
         <normaloff>:/icon/1106.jpg</normaloff>:/icon/1106.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="111.qrc">
         <normaloff>:/icon/111.png</normaloff>:/icon/111.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>300</width>
         <height>140</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="111.qrc"/>
 </resources>
 <connections/>
</ui>
